src/main/resources/userEmailTemplate-feedbackSessionUnpublished.html

```diff
@@ -0,0 +1,16 @@
+Hello ${userName},
```

Comment:
We need to add a test for the contents of this email template in `EmailGeneratorTest`